Pendulum/Data/PenPal+Extensions.swift

```diff
@@ -312,7 +312,7 @@ extension PenPal {
                 CNContactOrganizationNameKey,
                 CNContactImageDataAvailableKey,
                 CNContactThumbnailImageDataKey
-            ] as! [CNKeyDescriptor]
+            ] as [Any] as! [CNKeyDescriptor]
```

Comment:
What's necessitated adding this?

Reply:
As discussed, removed the change as only xcode 14.3.x seems to push for it.